### ISSUE TITLE
Clarify/better UX when trying to run multiple shaping commands

### ISSUE THREAD

**Issue by thehunmonkgroup** [yassen-itlabs/py-linux-traffic-control#3]
Trying to run a shaping command when one has already been run results in a cryptic error:
```
root@webrtc-test-0.stirlab.net$/var/local/git/py-linux-traffic-control: pyltc tc --iface eth0 -v --dclass tcp:8080-8080:512kbit
Namespace(clear=False, dclass=['tcp:8080-8080:512kbit'], iface='eth0', ingress=None, sclass=None, subparser='tc', verbose=True, version=False)
--> 'tc qdisc add dev eth0 root handle 1:0 htb'
RTNETLINK answers: File exists (rc=2)
```
Guessing this means all class args need to be passed in one command? If so, then it would be helpful to clarify this in the usage doc.

Also, a more user-friendly error message than ```RTNETLINK answers: File exists (rc=2)``` if the user does try to run a second shaping command.

**Comment by yassen-itlabs** [comment]
> it would be helpful to clarify this in the usage doc.
-- fixed with 5148fad79b.

>  a more user-friendly error message than RTNETLINK answers: File exists (rc=2)
There are numerous such errors and they may mean different things in different context. I suggest that we create a "dictionary" for these, adding descriptions of errors for the future generations ;) An ``--errors`` switch may be added later that prints that file for convenient access. When an error like that pops up, we can add ``(use --errors switch for possible reasons for this error)``.

**Comment by yassen-itlabs** [comment]
@thehunmonkgroup: I suggest that we devote a section in the documentation where probable causes for given kernel message. Let me know your thoughts on this.

**Comment by thehunmonkgroup** [comment]
Yes, this is a reasonable solution as opposed to trying to handle it all in code. Go for it!
